tensorboard/components/tf_paginated_view/tf-dom-repeat.html

```diff
@@ -166,6 +176,14 @@
         this._domBootstrapped = true;
       },
 
+      _updateActive() {
+        if (!this._domBootstrapped) return;
+
+        Array.from(this._renderedTemplateInst.values()).forEach((inst) => {
+          inst.notifyPath('active', this._contentActive, true /* fromAbove */);
```

Comment:
I’m trying to find out what `fromAbove` does, but I couldn’t find any
actual API docs for `notifyPath`, and what I presume is the definition
only takes `path` and `props`:

https://github.com/Polymer/polymer/blob/54da98b18b3b613c9fcc400bd8f8590a1ae6a303/lib/mixins/property-effects.html#L275

I also don’t see any 3-argument `notifyPath` invocations in the
implementation of `dom-repeat`.

Where is this defined or documented? How can I learn more about it?

Reply:
Per discussion: looks like this is a Polymer 1.x-ism.